### ISSUE TITLE
order translations by translator

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#2408]
Currently when there are multiple translators, we give segmented texts first, then legacy. Within each group no explicit ordering is given. Normally this is fine, but when "dense" view is on, we sometimes see different translators randomly. This is inconsistent, and it risks promoting poorer quality translations.

![Screenshot from 2022-02-25 06-39-30](https://user-images.githubusercontent.com/6112010/155595485-c928d57d-9ac3-49c1-bc11-b53569bc8bf5.png)

Let us add a config option so that we can determine which translation has priority.

sc-data/addition-info/prioritize.json

```
[
  {
    "translation-lang": "en",
    "root-lang": "pli",
    "tree": "sutta",
    "creator": "sujato"
  },
  {
    "translation-lang": "en",
    "root-lang": "pli",
    "tree": "vinaya",
    "creator": [
      "brahmali",
      "sujato"
    ]
  },
  {
    "translation-lang": "en",
    "root-lang": "lzh",
    "tree": "sutta",
    "creator": "patton"
  }
]
```

`tree` may be any UID found in the `tree` files: prioritize anything in that scope.

In plain English, this config file says:

>- Prioritize translations of Suttas from Pali to English by sujato. 
>- Prioritize translations of Vinaya from Pali to English by brahmali, then sujato. 
>- Prioritize translations of suttas from Chinese to English by patton.
